Algorithm/Examples/BubbleAlgorithm.cs

```diff
@@ -46,7 +46,9 @@ public class BubbleAlgorithm : QCAlgorithm
         Dictionary<string, RelativeStrengthIndex> rsiDic = new Dictionary<string, RelativeStrengthIndex>();
         Dictionary<string, MovingAverageConvergenceDivergence> macdDic = new Dictionary<string, MovingAverageConvergenceDivergence>();
 
-
+		/// <summary>
+		/// Initialise the Algorithm and Prepare Required Data.
```

Comment:
It may line up in your text editor due to a difference in the tab space settings used.